mr_project/mr_app/views.py

```diff
@@ -1,9 +1,231 @@
+
+
+def add_users(driver_arg, users_list):
+    n_users = len(users_list)
+    for i in range(n_users):
+        spawn_user_node(driver_arg, users_list[i])
+
+
+@csrf_exempt
+def get_user_emotion(request):
+    if request.method == 'GET':
+        query_arguments = request.GET
+        user = ''
+        emotion = ''
+
+        try:
+            user = query_arguments['user']
+            emotion = query_arguments['emotion']
+        except KeyError:
+            return HttpResponseBadRequest('Missing required key')
+
+        if check_for_user(DRIVER, user) and check_for_emotion(DRIVER, emotion):
+            epoch_dict = fetch_epochs_of_user_emotion_pair(DRIVER, user, emotion)
+
+            return JsonResponse(epoch_dict)
+        return HttpResponseBadRequest('Slow down dawg! The user and/or emotion do not exist, ya need a spelling lesson?')
+
+    return HttpResponseNotFound()
+
+
+@csrf_exempt
+def get_all_user_emotions(request):
+    if request.method == 'GET':
+        query_arguments = request.GET
+        user = ''
+
+        try:
+            user = query_arguments['user']
+        except KeyError:
+            return HttpResponseBadRequest('Missing required key')
+
+        if check_for_user(DRIVER, user):
+            all_epochs_dict = fetch_all_emotions_and_epochs_of_user(DRIVER, user)
+
+            return JsonResponse(all_epochs_dict)
+        return HttpResponseBadRequest('Slow down dawg! The user does not exist, ya need a spelling lesson?')
+
+    return HttpResponseNotFound()
+
+
+@csrf_exempt
+def create_user_emotion(request):
+    if request.method == 'POST':
+        body = json.loads(request.body)
+        user = ''
+        emotion = ''
+
+        try:
+            user = body['user']
+            emotion = body['emotion']
+        except KeyError:
+            return HttpResponseBadRequest('Missing required key')
+
+        if check_for_user(DRIVER, user) and check_for_emotion(DRIVER, emotion):
+            register_emotion(DRIVER, user, emotion)
+        elif check_for_user(DRIVER, user):
+            spawn_emotion_node(DRIVER, emotion)
+            register_emotion(DRIVER, user, emotion)
+        elif check_for_emotion(DRIVER, emotion):
+            spawn_user_node(DRIVER, user)
+            register_emotion(DRIVER, user, emotion)
+        else:
+            spawn_user_node(DRIVER, user)
+            spawn_emotion_node(DRIVER, emotion)
+            register_emotion(DRIVER, user, emotion)
+        return HttpResponse(status=codes.created)
+
+    return HttpResponseNotFound()
+
+
+@csrf_exempt
+def index(request, user_id, emotion):
+    if request.method == 'POST':
+        body = json.loads(request.body)
+        asdf = body['asdf']
+        return HttpResponse(f'you sent {asdf}')
 
-from django.http import HttpResponse
+    if request.method == 'GET':
+        print(request.GET)
+        return HttpResponse(f'you sent user_id = {user_id} and emotion = {emotion}')
 
+    return HttpResponseNotFound()
```

Comment:
This was just for testing, so let's just change this back to a basic hello world method until we have a frontend we can render.